extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -109,6 +109,7 @@ define('ext.wikia.adEngine.adContext', [
 			group.indexOf('AD_MIX_') === 0 &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)